internal/uvm/create_test.go

```diff
@@ -20,7 +20,7 @@ func TestCreateBadBootFilesPath(t *testing.T) {
 func TestCreateWCOWBadLayerFolders(t *testing.T) {
 	opts := NewDefaultOptionsWCOW(t.Name(), "")
 	_, err := CreateWCOW(context.Background(), opts)
-	if err == nil || (err != nil && err.Error() != `at least 2 LayerFolders must be supplied`) {
+	if err == nil || (err != nil && err.Error() != `UVM options incorrect or unsupported: at least 2 LayerFolders must be supplied`) {
```

Comment:
I'm fine with us making constants for things like `UVM options incorrect or unsupported:` and `at least 2 LayerFolders must be supplied` since we might want the ability to update those error messages without say breaking any tests.

Reply:
Ahh shoot I forgot to take this out

Reply:
Or I might be misunderstanding what you're talking about, you'd like if they were constants?

Reply:
Did you mean to take this out?

Reply:
Ahh no, I had a brainfart and forgot what this was for I apologize. This was added as if there was an error in **verifyOptions** CreateWCOW/LCOW will return an error with that error embedded. https://github.com/microsoft/hcsshim/blob/770d11942b76681c62bef58db8dae9cee9001b7b/internal/uvm/create_lcow.go#L195

Reply:
But in regards to your comment, the latter half of that was in the error constants that I just removed last push.

Reply:
We also have a errNotSupported constant but it's pretty generic. 
```go 
var errNotSupported = fmt.Errorf("not supported")
```

Reply:
What about something like 

`var errBadUVMOptions = errors.New("UVM options incorrect or unsupported")`
and
`return nil, fmt.Errorf(err, errBadUVMOptions)`

Thoughts?

Reply:
Yea thats fine with me, I'll add

Reply:
@katiewasnothere PTAL

Reply:
Oop never mind something failed lol